packages/shared/src/components/Icon.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
 
-type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
+type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxxlarge';
 
 const IconSize: Record<Size, string> = {
   xsmall: 'w-3 h-3',
   small: 'w-5 h-5',
   medium: 'w-6 h-6',
   large: 'w-8 h-8',
   xlarge: 'w-10 h-10',
+  xxxlarge: 'w-16 h-16',
```

Comment:
I see we will have a conflict of sizes - my PR uses w-14 h-14 as xxxlarge (and xxlarge is also used)

Reply:
Do you think I can safely add one bigger size?

Reply:
Yes xxxxlarge is completely safe, but name becomes a little bit weird (hope we are not going to add more sizes in the future)

Reply:
We can raise it with product after this to see if we can merge the xxx and xxxx variants as they are probably meant to be the same 👀

I'll raise it while still pushing this.

Reply:
Sounds good, let's do this